website/content/docs/v0.12/Reference/configuration.md

```diff
@@ -5305,3 +5331,50 @@ contents: /etc/kubernetes/auth
 
 
 
+
+## UdevConfig
+UdevConfig describes how the udev system should be configured.
+
+Appears in:
+
+
+- <code><a href="#machineconfig">MachineConfig</a>.udev</code>
+
+
+``` yaml
+# List of udev rules to apply to the udev system
+rules:
+    - SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="44", MODE="0660"
+```
+
+<hr />
+
+<div class="dd">
+
+<code>rules</code>  <i>[]string</i>
+
+</div>
+<div class="dt">
+
+List of udev rules to apply to the udev system
+
+
+
+Examples:
+
+
+``` yaml
+rules:
+    # List of udev rules to apply to the udev system
+    rules:
+        - SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="44", MODE="0660"
```

Comment:
hmm...did i goof something up? why is this `rules` > `rules` > `[]string`?

Reply:
figured it out...i'm just gonna remove the example. its not really needed since there is an example right above it.